### PR TITLE
[20.0-testing] Dicom archive: update test plan and change Series UI to SeriesUID

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
This pull request updates the test plan of the dicom archive module and changes the filter tag Series UI to become Series UID

### CODE REVIEW

File: modules/dicom_archive/test/TestPlan [hunk 1]
```diff
@@ -20,15 +20,17 @@
...
     [Manual Testing]
 7-  Under 'View Details' Click on 'Acquisition ID' to view the violated scans (if there are any) if permission 
-'Violated Scans: View all-sites Violated Scans' is granted
+    'Violated Scans: View all-sites Violated Scans' is granted
     [Manual Testing]
-8-  Under 'View Details' Click on 'Show/Hide Files' to view a list of all Dicom Files for the given patient
+8-  Under 'View Details' Click on 'Show/Hide Series' and 'Show/Hide Files' to view a list of all Dicom Series/Files,
+    respectively, for the given patient
     [Manual Testing]
-9- Under 'View Details' Click on 'Show/Hide metadata' to view the details of the metadata file
```
Comment (MounaSafiHarab) [inline modules/dicom_archive/test/TestPlan]: as far as I know, the show/hide metadata is not there, can someone @cmadjar @driusan confirm?

Comment (cmadjar) [reply modules/dicom_archive/test/TestPlan]: @MounaSafiHarab I confirmed with Dave. It was removed but the test plan was never updated. Thanks for doing it!!

File: modules/dicom_archive/test/TestPlan [hunk 2]
```diff
@@ -20,15 +20,17 @@
 6-  Under 'View Details' Check to make sure all the specific parameters are Populated
     [Manual Testing]
 7-  Under 'View Details' Click on 'Acquisition ID' to view the violated scans (if there are any) if permission 
-'Violated Scans: View all-sites Violated Scans' is granted
+    'Violated Scans: View all-sites Violated Scans' is granted
     [Manual Testing]
-8-  Under 'View Details' Click on 'Show/Hide Files' to view a list of all Dicom Files for the given patient
+8-  Under 'View Details' Click on 'Show/Hide Series' and 'Show/Hide Files' to view a list of all Dicom Series/Files,
```
Comment (cmadjar) [inline modules/dicom_archive/test/TestPlan]: DICOM instead of Dicom

File: modules/dicom_archive/test/TestPlan [hunk 3]
```diff
@@ -20,15 +20,17 @@
...
+    respectively, for the given patient
```
Comment (cmadjar) [inline modules/dicom_archive/test/TestPlan]: for the given **DICOM study** instead of patient

File: modules/dicom_archive/test/TestPlan [hunk 4]
```diff
@@ -20,15 +20,17 @@
...
     [Manual Testing]
-9- Under 'View Details' Click on 'Show/Hide metadata' to view the details of the metadata file
+9-  Make sure that the value of 'Patient ID' column is 'INVALID-HIDDEN' if the patient-name does not match the
```
Comment (cmadjar) [inline modules/dicom_archive/test/TestPlan]: I would rephrase this point like that:

Make sure that the value of **the** 'Patient ID' column is set to 'INVALID-HIDDEN' if the **Patient ID** does not match the regex in the Configuration module under the 'Imaging Modules' submenu. **Repeat the same operation for 'Patient Name'.**

File: modules/dicom_archive/test/TestPlan [hunk 5]
```diff
@@ -20,15 +20,17 @@
...
+    regex in the Configuration module under the 'Imaging Modules' submenu 
     [Manual Testing]
-10- Make sure that the value of 'Patient ID' column is 'INVALID-HIDDEN' if the patient-name does not match the 
-regex in the config.xml
+10- Click on 'View Images' to acess the Imaging-Browser for the given subject if 'View all-sites Imaging Browser 
+    pages' or 'View own-site Imaging Browser pages' permission is set
     [Manual Testing]
-11- Click on 'View Images' to acess the Imaging-Browser for the given subject if 'View all-sites Imaging Browser 
-pages' or 'View own-site Imaging Browser pages' permission is set
+11- Ensure that clicking on any row in the 'Archive Location` column triggers a download of the corresponding
```
Comment (cmadjar) [inline modules/dicom_archive/test/TestPlan]: weird `'` after ArchiveLocation

File: modules/dicom_archive/test/TestPlan [hunk 6]
```diff
@@ -20,15 +20,17 @@
...
     [Manual Testing]
-10- Make sure that the value of 'Patient ID' column is 'INVALID-HIDDEN' if the patient-name does not match the 
-regex in the config.xml
+10- Click on 'View Images' to acess the Imaging-Browser for the given subject if 'View all-sites Imaging Browser 
+    pages' or 'View own-site Imaging Browser pages' permission is set
     [Manual Testing]
-11- Click on 'View Images' to acess the Imaging-Browser for the given subject if 'View all-sites Imaging Browser 
-pages' or 'View own-site Imaging Browser pages' permission is set
+11- Ensure that clicking on any row in the 'Archive Location` column triggers a download of the corresponding 
+    archived DICOM
```
Comment (cmadjar) [inline modules/dicom_archive/test/TestPlan]: archived DICOM **study**

**Comment by MounaSafiHarab** [comment]
@cmadjar 
your comments are addressed. Thanks!

File: modules/dicom_archive/test/TestPlan [hunk 7]
```diff
@@ -20,15 +20,18 @@
...
     [Manual Testing]
-10- Make sure that the value of 'Patient ID' column is 'INVALID-HIDDEN' if the patient-name does not match the 
-regex in the config.xml
+10- Click on 'View Images' to acess the Imaging-Browser for the given subject if 'View all-sites Imaging Browser 
+    pages' or 'View own-site Imaging Browser pages' permission is set
     [Manual Testing]
-11- Click on 'View Images' to acess the Imaging-Browser for the given subject if 'View all-sites Imaging Browser 
-pages' or 'View own-site Imaging Browser pages' permission is set
+11- Ensure that clicking on any row in the 'Archive Location' column triggers a download of the corresponding 
+    archived DICOM study
```
Comment (PapillonMcGill) [inline modules/dicom_archive/test/TestPlan]: Only Item 3 as an ending dot. Suggest adding ending dot to each item for consistency (or remove ending dot at item 3).

Comment (MounaSafiHarab) [reply modules/dicom_archive/test/TestPlan]: strong eyes! thanks @PapillonMcGill ; removed the extra dot from items 3

File: modules/dicom_archive/test/TestPlan [hunk 8]
```diff
@@ -20,15 +20,18 @@
...
-'Violated Scans: View all-sites Violated Scans' is granted
+    'Violated Scans: View all-sites Violated Scans' is granted
     [Manual Testing]
-8-  Under 'View Details' Click on 'Show/Hide Files' to view a list of all Dicom Files for the given patient
+8-  Under 'View Details' Click on 'Show/Hide Series' and 'Show/Hide Files' to view a list of all DICOM Series/Files,
+    respectively, for the given DICOM study
     [Manual Testing]
-9- Under 'View Details' Click on 'Show/Hide metadata' to view the details of the metadata file
+9-  Make sure that the value of 'Patient ID' column is 'INVALID-HIDDEN' if the PatientID does not match the 
+    regex in the Configuration module under the 'Imaging Modules' submenu. Repeat the same operation for
```
Comment (cmadjar) [inline modules/dicom_archive/test/TestPlan]: ...regex **set** in the Configuration...

Comment (cmadjar) [reply modules/dicom_archive/test/TestPlan]: @MounaSafiHarab one final comment.

Comment (MounaSafiHarab) [reply modules/dicom_archive/test/TestPlan]: done!
